src/utils/curriculum/features.ts

```diff
@@ -30,16 +30,19 @@ export function getUnitFeatures(unit?: Unit | null) {
       subjectcategories: {
         all_disabled: true,
         default_category_id: 4,
+        group_by_subjectcategory: true,
       },
     };
   } else if (
     unit.subject_slug === "english" &&
-    unit.phase_slug === "secondary"
+    unit.phase_slug === "secondary" &&
+    ["10", "11"].includes(unit.year)
```

Comment:
@orangemug Is there a practical/logical reason why we're not using key stage here? i.e. years 10 and 11 = KS4, so the if statement would be slightly less messy if you could use that as the selection criteria. See also lines 20-21 above.

Reply:
Made these changes in the latest commits